CHANGELOG.md

```diff
@@ -1,7 +1,49 @@
 # Change Log
 All notable changes to the "vscode-azurearmtools" extension will be documented in this file.
 
-## Version 0.8.0 (2019-10-01)
+## Coming Soon
+- Support for variable iteration ("COPY blocks")
```

Comment:
I've never seen a "coming soon" section in a changelog before...

Reply:
Me neither :-), but AlexF was talking about putting roadmaps there.  Was going to see what he thought.